tests/test_target/test_binning.py

```diff
@@ -61,3 +61,15 @@ def test_pandas_bins(self):
         visualizer.fit(y)
         visualizer.finalize()
         self.assert_images_similar(visualizer, tol=0.5)
+    
+    def test_quick_method(self):
+        """
+        Test the quick method with producing a valid visualization
+        """     
+        data = load_occupancy(return_dataset=True)
+        _, y = data.to_numpy()
+        
+        visualizer = balanced_binning_reference(y, show=False)
+
+        assert isinstance(visualizer, BalancedBinningReference)
+        self.assert_images_similar(visualizer, tol=0.5)
```

Comment:
@wagner2010 @Kautumn06 my apologies if my previous comment caused confusion; I should have been more clear! 

When you skip a test, the `skip-if` decorator needs to go right on top of the function definition in order to work, e.g.:

```python
    @pytest.mark.skipif(pd is None, reason="pandas is required")
    def test_quick_method(self):
        """
        Test the quick method with producing a valid visualization
        """     
        data = load_occupancy(return_dataset=True)
        _, y = data.to_pandas()
        
        visualizer = balanced_binning_reference(y, show=False)

        assert isinstance(visualizer, BalancedBinningReference)
        self.assert_images_similar(visualizer, tol=0.5)
```

I like to think of decorators as sparkly tiaras that you put on top of the function or method's head so that they can say "do you even *know* who I am?"  👑 Then Python sees the tiara first and changes how it treats the function/method (in this case knowing to skip the whole test if there's no Pandas installed on that machine)   

Maybe @bbengfort and I can do a clinic on testing sometime... In the meantime though, you can read more about Python decorators [here](https://www.python.org/dev/peps/pep-0318/)). Thanks again for pushing ahead with this PR & review!

Reply:
Thanks for sharing the resource!  It helps us improve!